contrib/jenkins/build.sh

```diff
@@ -38,7 +38,7 @@ done
   || error_exit '--project is a required parameter'
 
 if [[ "$(uname -s)" == "Linux" ]]; then
-  GIT_HEAD="$(git describe --tags --always --abbrev=7 --dirty)"
+  GIT_HEAD="$(git describe --always --abbrev=7 --dirty)"
```

Comment:
The 2 commands have subtle differences. `git rev-parse --short HEAD` will always give us the latest sha, even if the tree is dirty. Whereas `git describe --tags --always --abbrev=7 --dirty` will append the suffix `-dirty` for dirty tree. It might not make much different for us currently, imo but it might be good to know that the tree is dirty in case we want to `sed` some files or modify the source code in our CI.